Resources/Prototypes/_EE/Contractors/passports.yml

```
@@ -142,3 +142,14 @@
       - state: passport_goldendeep_open
       - state: passport_species_ipc
+
+- type: entity
+  name: Qiimaeru passport
+  parent: BasePassport
+  id: QiimaeruPassport
+  components:
+    - type: Sprite
+      sprite: _EE/Contractors/Passport.rsi
+      layers:
+      - state: passport_qiimaeru_open
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)